src/screens/CollectionsListScreen.js

```diff
@@ -28,9 +31,10 @@ const CollectionsListScreen = ({
       <StatusBar backgroundColor={Colors.primary} barStyle="dark-content" />
       <Header style={styles.header}>
         <Left />
-        <Body>
+        <View style={styles.headerTitleContainer}>
+          <HeaderCountIcon count={collectionsCount} />
           <Title style={styles.headerText}>Collections</Title>
-        </Body>
+        </View>
```

Comment:
<img width="365" alt="image" src="https://user-images.githubusercontent.com/3383704/122102017-f7e1b780-cde2-11eb-9d7d-8fd1bd3a4737.png">

Although pluralization would be better handled with an i18n library, maybe a quick fix would be to have the count, afterward? eg: `Collections (1)`

Reply:
This would be a design decision